api/api.py

```diff
@@ -435,7 +435,7 @@ def add_fast_build() -> FastBuild:
     """Returns a FastBuild that is associated with the image that was built from a ``custom_build``. When the container needs to be rebuilt it will be built using the ``CustomBuild``. Otherwise update will be done with the ``FastBuild`` instructions. """
     pass
 
-def custom_build(ref: str, command: str, deps: List[str], tag: str = "", disable_push: bool = False, live_update: List[LiveUpdateStep]=[], entrypoint: str="") -> CustomBuild:
+def custom_build(ref: str, command: str, deps: List[str], tag: str = "", disable_push: bool = False, live_update: List[LiveUpdateStep]=[], ignore: Union[str, List[str]] = [], entrypoint: str="") -> CustomBuild:
```

Comment:
I think the variable is name is "ignores" rather than "ignore"

Reply:
hmm, [doesn't look like it](https://github.com/windmilleng/tilt/blob/71d688dd63b727a1a29fa0b0ad4e33cebf24f658/internal/tiltfile/docker.go#L242)?

Reply:
Oo ok I see- it looks like we call it "ignores" internally after we parse the args, my bad! Otherwise LGTM!